### ISSUE TITLE
Add Bunny Fonts support for GDPR compliance.

### ISSUE THREAD

**Issue by shoelaced** [soflyy/oxygen-bugs-and-features#3098]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*

Google Fonts has been blacklisted by the GDPR due to its tracking capabilities. Bunny Fonts offers a simple, compatible swap that is fully GDPR compliant: https://fonts.bunny.net/

It would be helpful to either replace Google Fonts or add it as an option to be used in lieu of Google Fonts.

**Comment by s-a-s-k-i-a** [comment]
Will this be done? Waiting for it too

**Comment by shoelaced** [comment]
Currently managing this with JS but yes, please consider implementing this directly!

```
document.querySelectorAll( 'link[href*="fonts.googleapis.com"]' ).forEach( function( link ) {
    link.href = link.href.replace( 'fonts.googleapis.com', 'fonts.bunny.net' );
} );
```

**Comment by MattHag** [comment]
I don't think this will be implemented as it's very specific, and I wouldn't want to rely on an external site for fonts anyways. There are already complaints, about broken fonts out there in the Oxygen group as well.

It would be much better for Oxygen to support local fonts, like the script from Matthias Altmann https://www.altmann.de/blog/code-snippet-eigene-schriftarten/ or a little better implementation than Bricks has.

**Comment by s-a-s-k-i-a** [comment]
We would need a filter function. easy to implement. I started an issue for an enhancement request here: #3502
